package.json

```diff
@@ -129,5 +129,15 @@
     "prepare": "lerna bootstrap --hoist",
     "contributors": "githubcontrib --owner transloadit --repo uppy --cols 6 $([ \"${GITHUB_TOKEN:-}\" == \"\" ] && echo \"\" || echo \"--authToken ${GITHUB_TOKEN}\") --showlogin true --sortOrder desc",
     "contributors:save": "replace-x -m '<!--contributors-->[\\s\\S]+<!--/contributors-->' \"<!--contributors-->\n## Contributors\n\n$(npm run --silent contributors)\n<!--/contributors-->\" README.md"
+  },
+  "jest": {
+    "automock": false,
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "packages/**/src/**"
+    ],
+    "testMatch": [
+      "**/packages/**/*.test.js"
+    ]
```

Comment:
Wouldn’t a separate [`jest.config.js`](https://jestjs.io/docs/en/configuration.html) be better since that’s how we are doing it with all other build tools?